src/Microsoft.ML.Data/Transforms/TermTransform.cs

```diff
@@ -226,70 +227,154 @@ private CodecFactory CodecFactory
         /// Public constructor corresponding to SignatureDataTransform.
         /// </summary>
         public TermTransform(IHostEnvironment env, Arguments args, IDataView input)
```

Comment:
>public TermTransform(IHostEnvironment env, Arguments args, IDataView input) [](start = 8, length = 75)

it doesn't correspond to the signature any more, right?

Reply:
Actually, I'd suggest having the 'training' happen in `public static TermTransform Train(env, args, trainData)`, and ctors should be private.

---
In reply to: [214131316](https://github.com/dotnet/machinelearning/pull/759#discussion_r214131316) [](ancestors = 214131316)